frontend/Panel.js

```diff
@@ -234,7 +239,16 @@ class Panel extends React.Component {
       );
     }
     if (!this.state.isReact) {
-      return <div style={styles.loading}><h2>Looking for React…</h2></div>;
+      return (
+        <div style={styles.loading}>
+          <h2 style={styles.errorMessage}>Unable to find React…</h2>
```

Comment:
I thought the code [here](https://github.com/Tushkiz/react-devtools/blob/195ecd46e2fb1fbbf1bf7afd392f8f53c06fd2ea/frontend/Panel.js#L209-L222) could distinguish loading vs failed state, but it seems that the failed state isn't actually 'failed'.

How about we reword the messages as
- Connecting to React…
- Looking for React…
  - Please check if the devtools extension has 'Allow access to file URLs' permission